.goreleaser-stable.yaml

```diff
@@ -64,6 +64,37 @@ nix:
     install: |-
       mkdir -p $out/bin
       cp -vr ./dist/kraft $out/bin/kraft
+brews:
+  - name: kraftkit
+    alternative_names:
+      - kraftkit@{{ .Version }}
+      - kraftkit@{{ .Major }}
+    url_template: "https://github.com/unikraft/kraftkit/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+    commit_author:
+      name: Unikraft Bot
+      email: monkey@unikraft.io
+    commit_msg_template: "feat(Formula): Bump {{ .ProjectName }} to version {{ .Tag }}"
```

Comment:
I think we can remove `feat(Formula)`, this repository will only ever be updated by goreleaser.